packages/codemirror-lang-dentaku/src/linting.ts

```diff
@@ -126,32 +143,43 @@ export function dentakuLinter({
           const functionName = view.state.sliceDoc(
             functionNameNode.from,
             functionNameNode.to
-          );
-          const expectedArgumentCount: number | undefined =
-            knownArgumentCounts[
-              functionName as keyof typeof knownArgumentCounts
-            ];
+          ) as BuiltInFunctionsType;
```

Comment:
issue: I think this is semantically incorrect because this function name might not be of a built-in function

suggestion: on the following line, I'd suggest writing this:
```ts
const { minArgs = 0, maxArgs = Infinity } = functionName in knownFunctions ? knownFunctions[functionName as keyof typeof knownFunctions] : {}
```

Reply:
That was a miss between adding the arity changes to the built-in functions and the custom functions. Fixing it.